packages/@ama-sdk/swagger-builder/src/core/swagger-spec-wrappers/swagger-spec-object.ts

```diff
@@ -37,26 +36,26 @@ export class SwaggerSpecObject implements SwaggerSpec {
 
   /** @inheritdoc */
   public async getDefinitions(): Promise<Record<string, Schema>> {
```

Comment:
I thought we had a rule to prevent using `async` keyword when there is no `await` in the function

Reply:
Well spotted!
We should indeed, it may have been removed because not part of "recommended".
The rule is [require-await](https://eslint.org/docs/latest/rules/require-await)